sklearn/decomposition/_pca.py

```diff
@@ -283,6 +283,8 @@ class PCA(_BasePCA):
     Series B (Statistical Methodology), 61(3), 611-622.
     <http://www.miketipping.com/papers/met-mppca.pdf>`_
     via the score and score_samples methods.
+    The explained variance according to the two is
+    explained_variance_ = (S ** 2) / n_samples
```

Comment:
The right place to put a comment is above, under the attribute section. There, one should just say that `n_samples - 1` degrees of freedoms are used for the variance estimation.